src/cfnlint/data/ExtendedSpecs/all/01_spec_patch.json

```diff
@@ -383,5 +383,25 @@
       "Required": true,
       "UpdateType": "Immutable"
     }
+  },
+  {
+    "op": "add",
+    "path": "/PropertyTypes/AWS::MediaPackage::PackagingConfiguration.CmafEncryption/Properties/SpekeKeyProvider/Type",
+    "value": "SpekeKeyProvider"
+  },
+  {
+    "op": "add",
+    "path": "/PropertyTypes/AWS::MediaPackage::PackagingConfiguration.DashEncryption/Properties/SpekeKeyProvider/Type",
+    "value": "SpekeKeyProvider"
+  },
+  {
+    "op": "add",
+    "path": "/PropertyTypes/AWS::MediaPackage::PackagingConfiguration.HlsEncryption/Properties/SpekeKeyProvider/Type",
+    "value": "SpekeKeyProvider"
+  },
+  {
+    "op": "add",
+    "path": "/PropertyTypes/AWS::MediaPackage::PackagingConfiguration.MssEncryption/Properties/SpekeKeyProvider/Type",
+    "value": "SpekeKeyProvider"
```

Comment:
calling these out as the manual patches for anyone looking to do similar in other repos